src/current/v23.2/node-shutdown.md

```diff
@@ -76,31 +89,35 @@ Node drain consists of the following consecutive phases:
 1. **Lease transfer phase:** The node's [`is_draining`]({% link {{ page.version.version }}/cockroach-node.md %}#node-status) field is set to `true`, which removes the node as a candidate for replica rebalancing, lease transfers, and query planning. Any [range leases]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) or [Raft leaderships]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) must be transferred to other nodes. This phase completes when all range leases and Raft leaderships have been transferred.
 
     <section class="filter-content" markdown="1" data-scope="decommission">
-    Since all range replicas were already removed from the node during the [decommissioning](#decommissioning) stage, this step immediately resolves.
+    Since all range replicas were already removed from the node during the [draining](#draining) stage, this step immediately resolves.
     </section>
 
 <section class="filter-content" markdown="1" data-scope="drain">
-When [draining manually](#drain-a-node-manually), if the above steps have not completed after 10 minutes by default, node draining will stop and must be restarted to continue. For more information, see [Drain timeout](#drain-timeout).
+When [draining manually](#drain-a-node-manually), if the above steps have not completed after [`server.shutdown.drain_wait`](#server-shutdown-drain_wait) (10 minutes by default), node draining will stop and must be restarted manually to continue. For more information, see [Drain timeout](#drain-timeout).
 </section>
 
 <section class="filter-content" markdown="1" data-scope="decommission">
 #### Status change
 
-After decommissioning and draining are both complete, the node membership changes from `decommissioning` to `decommissioned`.
+After the node is drained and decommissioned, the node membership status changes from `decommissioning` to `decommissioned`.
 
-This node is now cut off from communicating with the rest of the cluster. A client attempting to connect to a `decommissioned` node and run a query will get an error.
+This node is now prevented from communicating with the rest of the cluster, and client connection attempts to the node will result in an error.
 </section>
 
-At this point, the `cockroach` process is still running. It is only stopped by process termination.
+At this point, it is safe to terminate the `cockroach` process manually or using your process manager or other deployment tooling (such as Kubernetes, Nomad, or Docker).
 
 #### Process termination
 
 <section class="filter-content" markdown="1" data-scope="decommission">
-An operator [terminates the node process](#terminate-the-node-process).
+After draining and decommissioning are complete, an operator [terminates the node process](#terminate-the-node-process).
```

Comment:
this anchor link doesn't point to anything

https://deploy-preview-18424--cockroachdb-docs.netlify.app/docs/v23.2/node-shutdown.html#terminate-the-node-process